models/template/Dockerfile

```diff
@@ -0,0 +1,69 @@
+########################################################################
+# First we build the actual model, this has everything that is needed
+# for the model to compile. Next we create the final image that has the
+# PEcAn code as well as only the model binary.
+########################################################################
+
+# this needs to be at the top, what version are we building
+ARG IMAGE_VERSION="latest"
+
+# ----------------------------------------------------------------------
+# BUILD MODEL BINARY
+# ----------------------------------------------------------------------
+FROM debian:stretch as model-binary
+
+# Some variables that can be used to set control the docker build
+ARG MODEL_VERSION=git
+
```

Comment:
would we want an `ENV` for specifying compiler etc?

Reply:
I think people should modify the Dockerfile. Most models will need different libraries, compilers etc.

Reply:
I updated the README.md to hopefully clarify some of this.